community/modules/file-system/DDN-EXAScaler/variables.tf

```diff
@@ -189,25 +189,27 @@ variable "boot" {
   type = object({
     disk_type   = string
     auto_delete = bool
+    script_url  = string
   })
   default = {
     disk_type   = "pd-standard"
     auto_delete = true
+    script_url  = null
   }
 }
 
 # Source image properties
 # project: project name
-# name: image name
+# family: image family name
 variable "image" {
```

Comment:
A suggestion (feel free to disregard):
Take all of project, name and family as input but validate with an error if name is used, indicating how the interface has changed. That way, the error received if someone is still using the old format clearly indicated how to fix it, rather than a confusing "variable not found" error.

Reply:
I like this and will implement in a follow on PR.